test/setup/variables.tf

```diff
@@ -37,3 +37,9 @@ variable "example_foundations_mode" {
 variable "domain_to_allow" {
   description = "The test domain_to_allow allow users from in IAM."
 }
+
+variable "auto_create_network" {
+  description = "Control whether the 'default' network will be created in the project."
+  type        = bool
+  default     = true
+}
```

Comment:
this should not be a variable, it is a hard coded value when using the project factory method